ports/apr-util/portfile.cmake

```
@@ -1,10 +1,18 @@
 include(vcpkg_common_functions)
 
-vcpkg_download_distfile(ARCHIVE
-  URLS "https://archive.apache.org/dist/apr/apr-util-1.6.0-win32-src.zip"
-  FILENAME "apr-util-1.6.0-win32-src.zip"
-  SHA512 98679ea181d3132020713481703bbefa0c174e0b2a0df65dfdd176e9771935e1f9455c4242bac19dded9414abe2b9d293fcc674ab16f96d8987bcf26346fce3a
-)
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
+    vcpkg_download_distfile(ARCHIVE
+        URLS "https://archive.apache.org/dist/apr/apr-util-1.7.0-win32-src.zip"
```

```suggestion
        URLS "https://archive.apache.org/dist/apr/apr-util-${APR_UTIL_VERSION}-win32-src.zip"
```
Could you please `set(APR_UTIL_VERSION 1.7.0)` and use the variable here?